.github/workflows/pull_request.yml

```
@@ -24,18 +24,19 @@ jobs:
     steps:
       - name: Support long paths
         if: ${{ matrix.os == 'windows-latest' }}
-        uses: kiegroup/droolsjbpm-build-bootstrap/.ci/actions/long-paths@main
+        uses: kiegroup/kie-ci/.ci/actions/long-paths@main
       - name: Java and Maven Setup
-        uses: kiegroup/droolsjbpm-build-bootstrap/.ci/actions/maven@main
+        uses: kiegroup/kie-ci/.ci/actions/maven@main
         with:
           java-version: ${{ matrix.java-version }}
           maven-version: ${{ matrix.maven-version }}
           cache-key-prefix: ${{ runner.os }}-${{ matrix.java-version }}-maven${{ matrix.maven-version }}
       - name: Build Chain
-        uses: kiegroup/droolsjbpm-build-bootstrap/.ci/actions/build-chain@main
+        uses: kiegroup/kie-ci/.ci/actions/build-chain@main
         with:
+          definition-file: https://raw.githubusercontent.com/${GROUP:kiegroup}/kogito-pipelines/${BRANCH:main}/.ci/pull-request-config.yaml
```

Drools `7.67.x` is related to RHPAM, therefore we should take definition file from `droolsjbpm-build-bootstrap` to keep it exactly as it was.
```suggestion
          definition-file: https://raw.githubusercontent.com/${GROUP:kiegroup}/droolsjbpm-build-bootstrap/${BRANCH:main}/.ci/pull-request-config.yaml
```

---

I agree @lampajr